config/jobs/kubernetes-csi/csi-lib-utils/csi-lib-utils-config.yaml

```diff
@@ -0,0 +1,17 @@
+presubmits:
+  github.com/kubernetes-csi/csi-lib-utils:
```

Comment:
ooooops, drop github.com here 😂

Reply:
Thanks for debugging it. Fix is in https://github.com/kubernetes/test-infra/pull/10340.